packages/terra-search-field/src/terra-dev-site/doc/example/SearchFieldDisableAutoSearch.jsx

```diff
@@ -1,11 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
-import SearchFieldExampleTemplate from 'terra-search-field/lib/terra-dev-site/doc/example/SearchFieldExampleTemplate';
+import SearchField from 'terra-search-field/lib/SearchField';
 
-const SearchFieldDisableAutoSearch = () => (
-  <SearchFieldExampleTemplate
-    disableAutoSearch
-  />
-);
+const INVALID_MESSAGE = 'The default minimum search length is 2.';
+
+const SearchFieldDisableAutoSearch = () => {
+  const [searchText, setSearchText] = useState('');
+
+  const message = searchText.length >= 2 ? `Search text: ${searchText}` : INVALID_MESSAGE;
+
+  return (
+    <div>
+      <p>{message}</p>
+      <SearchField disableAutoSearch onSearch={setSearchText} onInvalidSearch={setSearchText} />
```

Comment:
Even for this example <SearchField DisableAutoSearch /> will be fine.

Reply:
Updated this as well.